public/css/docsearch.css

```diff
@@ -0,0 +1,34 @@
+/* input.docsearch {} */
+
+/* Main dropdown wrapper */
+.algolia-autocomplete .ds-dropdown-menu {
+  width: 500px;
+}
+
+/* Main category (eg. Getting Started) */
+.algolia-autocomplete .algolia-docsearch-suggestion--category-header {
+  color: darkgray;
+  border: 1px solid gray;
+}
```

Comment:
What about making the CSS:

```css
.algolia-autocomplete .ds-dropdown-menu .ds-suggestions a {
  text-decoration: none;
}

/* Main category (eg. Getting Started) */
.algolia-autocomplete .algolia-docsearch-suggestion--category-header {
  color: var(--g8);
  background-color: var(--g1);
  border-left: 8px solid var(--g3);
  padding: 1rem !important;
}
```

which makes the header look like this:

![Screen Shot 2021-09-23 at 16 48 13](https://user-images.githubusercontent.com/353180/134581719-b2d6ddf0-28ba-4466-a170-68f3b2da5266.png)

to tie the search dialog look and feel closer to the rest of the site.

Reply:
That's cool. looks more integrated that way.